allink_core/core_apps/allink_content/cms_plugins.py

```diff
@@ -99,55 +101,67 @@ class CMSAllinkContentPlugin(AllinkMediaAdminMixin, CMSPluginBase):
     child_classes = ['ContentColumnPlugin']
     form = AllinkContentPluginForm
 
-    fieldsets = (
-        (None, {
-            'fields': (
-                'title',
-                'title_size',
-                'template',
-            ),
-        }),
-        ('Section Options', {
-            'classes': ('collapse',),
-            'fields': [
-                'container_enabled',
-                'inverted_colors_enabled',
-                'overlay_enabled',
-                'bg_color',
-            ]
-        }),
-        ('Spacings', {
-            'classes': ('collapse',),
-            'fields': [
-                'project_css_spacings_top_bottom',
-                'project_css_spacings_top',
-                'project_css_spacings_bottom',
-            ]
-        }),
-        ('Background Image (full width)', {
-            'classes': ('collapse',),
-            'fields': (
-                'bg_image_outer_container',
-                'dynamic_height_enabled',
-            )
-        }),
-        ('Background Video (Important: Only works if all fields are set)', {
-            'classes': ('collapse',),
-            'fields': (
-                'video_file',
-                'video_poster_image',
-                'video_mobile_image',
+    def get_fieldsets(self, request, obj=None):
+        """
+        CONTENT_REDUCED_FEATURE_SET = True will remove the following fields:
+        'bg_image_outer_container', 'video_file', 'video_poster_image', 'video_mobile_image'
+        """
+
+        fieldsets = (
+            (None, {
+                'fields': (
+                    'title',
+                    'title_size',
+                    'template',
+                ),
+            }),
+            ('Section Options', {
+                'classes': ('collapse',),
+                'fields': [
+                    'container_enabled',
+                    'inverted_colors_enabled',
+                    'overlay_enabled',
+                    'bg_color',
+                ]
+            }),
+            ('Spacings', {
+                'classes': ('collapse',),
+                'fields': [
+                    'project_css_spacings_top_bottom',
+                    'project_css_spacings_top',
+                    'project_css_spacings_bottom',
+                ]
+            })
+        )
+        
+        if not getattr(settings, 'CONTENT_REDUCED_FEATURE_SET', False):
```

Comment:
I would call the constant CONTENT_EXTENDED_FEATURE_SET because we set it to false by default.